hardhat.config.js

```diff
@@ -1,17 +1,22 @@
 require("@nomiclabs/hardhat-waffle");
 require('dotenv').config();
+require('@openzeppelin/hardhat-upgrades');
 
-const { INFURA_API_KEY } = process.env;
+const { INFURA_API_KEY, GOERLI_PRIVATE_KEY, MAINNET_PRIVATE_KEY } = process.env;
 
 module.exports = {
-  defaultNetwork: 'hardhat',
-  solidity: "0.8.10",
+  solidity: '0.8.15',
   networks: {
-    hardhat: {
-      forking: {
-        url: `https://mainnet.infura.io/v3/${INFURA_API_KEY}`,
-      },
-      chainId: 1559
+    hardhat: {},
+    goerli: {
+      url: `https://goerli.infura.io/v3/${INFURA_API_KEY}`,
+      accounts: [`${GOERLI_PRIVATE_KEY}`],
+      // gasMultiplier: 2,
+    },
+    mainnet: {
+      url: `https://mainnet.infura.io/v3/${INFURA_API_KEY}`,
+      // accounts: [`${MAINNET_PRIVATE_KEY}`],
+      // gasMultiplier: 2,
```

Comment:
keep comment if we would need to uncomment as some point, if it's redundant let's remove it

Reply:
Saving for longevity otherwise it'll be squashed